mayuctl/config.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"fmt"
+	"os"
+	"github.com/giantswarm/mayu/pxemgr"
+	"io/ioutil"
+)
+
+type ConfigFlags struct {
+	Method string
+	ConfigFile string
+}
+
+var (
+	configCmd = &cobra.Command{
+		Use:   "config get",
+		Short: "fetch or change curent mayu config",
+		Long:  "fetch or change curent mayu config",
+		Run:   configRun,
+	}
+	configFlags = &ConfigFlags{}
+)
+
+func init() {
+	configCmd.PersistentFlags().StringVar(&configFlags.Method, "method", "get","Method - 'get' or 'set', default is get. Set method requires config-file flag.")
```

Comment:
Is it a conscious decision to have the actual action being a flag? I would like to align this with `gsctl`, where we have something like `gsctl create cluster`. Here for instance it would then be `mayuctl set config`. I understood we want to go with mayu in the longer term, so giving it more love would be really cool. Maybe @marians also has some opinion on this UX wise.